nebula/ui/components/MZStepNavigation.qml

```diff
@@ -0,0 +1,104 @@
+import QtQuick 2.5
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import Mozilla.Shared 1.0
+
+//Usage: pass views to this component using views property
+//Note: views must be passed in order that they will appear within the navigation
+//Note: Glitchy when used inside MZViewBase and does not consider the navbar
```

Comment:
Is this indicating that the navigation component shouldn't currently be used inside MZViewBase? Consider addding a few more details.